tests/testthat/test-utilities-simulation.R

```diff
@@ -22,15 +22,17 @@ test_that("`initializeSimulation()` does not fail when additionalParams is empty
 
 ## context("compareSimulationParameters")
 
-test_that("`compareSimulationParameters()` produces no differences with identical simulations", {
+test_that("`compareSimulations()` produces no differences with identical simulations", {
```

Comment:
Maybe add a test case for when simulations contains different paths and/or paramter values ?

Reply:
@Felixmil Added.